workshops/pacman game/README.md

```diff
@@ -0,0 +1,337 @@
+---
+name: 'Pac-man Game'
+description: 'Simple Pac-man game using python'
+author: '@YashKalbande'
+img: 'https://cloud-k9b5z9yni.vercel.app/1pac-man.png'
+---
+
+![pacman gif](https://cloud-k9b5z9yni.vercel.app/0pacman.gif)
+
+## Introduction
+
+[Pac-man](https://en.wikipedia.org/wiki/Pac-Man) is a famous video game featuring a yellow character roaming around a maze collecting different things along the way. In this workshop, you'll learn how to create your own Pac-man game using the [Python](https://en.wikipedia.org/wiki/Python_(programming_language)) [Freegames](https://pypi.org/project/freegames/) library.
+
+Here's a demo of what we'll be making. You can see both the final code and test it out on this Repl environment: https://repl.it/@YashKalbande/Easy-Level
+
+* **NOTE:** If you ever get lost, feel free to open the Repl above and see where the code is being placed. All the code explained in this workshop is in order of first line to last line :) - Happy hacking and good luck!
+
+We're going to be coding this using a principle called [modular programming](https://en.wikipedia.org/wiki/Modular_programming). This means we're going to be writing different functions that we will call at the end. A function is written by putting the function name, what parameters it needs, and then what happens with the parameters. For example, this is how I will write a function:
+
+```python
+functionName(parameter1, parameter2):
+	total = parameter1 + parameter 2
+  return total
+```
+
+Now, whenever I call `functionName`, I will get the sum of it's two parameters. You can call a function by just filling in the parameters:
+
+```python
+sum = functionName(5, 6)
+print(sum) //prints 11
+```
+
+If at any time you want to test the functions we create, just use the example above as a reference and write it in a new line.
```

Comment:
Not sure what this sentence is saying. Clarify maybe, or maybe just remove it altogether